app/templates/play/campaign-view.jade

```diff
@@ -148,7 +148,10 @@ if view.showAds()
                   button.btn.btn-primary.btn.btn-lg.btn-block.btn-illustrated
                     span(data-i18n="common.play") Play
               else
-                button.btn.btn-success.btn.btn-lg.btn-illustrated.start-level(data-i18n="common.play") Play
+                if level.requiresSubscription && !me.isPremium()
+                  button.btn.btn-primary.btn.btn-lg.btn-illustrated.start-level(data-i18n="common.subscribe_to_play") Subscribe to Play
```

Comment:
I usually remove the static english here if there's data-i18n too.  I think it might serve as a fallback and if your i18n is incorrect, which means translations won't work.  No need to change for this PR.